### DISCUSSION TITLE
Possibilité de modifier une caractéristique d'un joueur sans passer par une épreuve ou un loot

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#64] [Boite à idées]
Il faudrait pouvoir augmenter/baisser les caractéristiques (esprit/corps, mais surtout points de vie) sans passer par épreuve/loot.
Pourrait être utile par exemple pour rendre de la vie à un personnage qui boit une potion de soin.
Actuellement, c'est faisable en demandant une épreuve très difficile au joueur et en mettant un bonus à la stat en cas d'échec de l'épreuve, mais il faut s'y reprendre à plusieurs fois.
C'est aussi faisable en accordant un loot, mais il y a sûrement un nombre de loots maximum qu'un personnage peut avoir ?

**Comment by uurshin** [comment]
Comme répondu ailleurs, pas de max de loot.

J'aimerai bien rajouter dans une future version un certain nombre d'opérations qu'on pourrait faire en cliquant sur un PJ, notamment : 
- modifier les PV
- modifier les tags
- modifier les caracs
- supprimer

**Reply by ElethornGM** [reply]
Super merci
Ok c'est clair
